README.md

```diff
@@ -25,7 +31,9 @@ Quickly build beautiful [React](https://reactjs.org/) apps. MUI is a simple and
 
 ## Installation
 
-MUI is available as an [npm package](https://www.npmjs.com/package/@mui/material).
+### Material UI
+
+Material UI is available as an [npm package](https://www.npmjs.com/package/@mui/material).
 
 **[Stable channel v5](https://mui.com/)**
```

Comment:
I think that we could move this out of Material UI. We have one version, and it's for MUI Core, not Material UI.